src/System.Private.ServiceModel/tests/Common/src/BridgeClient.cs

```diff
@@ -58,7 +65,18 @@ private static void MakeConfigRequest()
                 {
                     var response = httpClient.PostAsync("/config/", content).Result;
                     if (!response.IsSuccessStatusCode)
-                        throw new Exception("Unexpected status code: " + response.StatusCode);
+                    {
+                        string reason = String.Empty;
+                        if (response.Content != null)
+                        {
+                            string contentAsString = response.Content.ReadAsStringAsync().Result;
+                            reason = String.Format("{0}Bridge returned content:{0}{1}",
+                                                    Environment.NewLine, contentAsString);
+                        }
+                        throw new Exception(
+                            String.Format("Bridge returned unexpected status code {0}{1}", 
+                                            response.StatusCode, reason));
+                    }
```

Comment:
There's also a status reason which is a textual description which might be helpful to include in the description.